docs/dev/developer-guides/aa.md

```diff
@@ -15,15 +12,11 @@ The former type is the only one that can initiate transactions,
 while the latter is the only one that can implement arbitrary logic. For some use-cases, like smart-contract wallets or privacy protocols, this difference can create a lot of friction.
 As a result, such applications require L1 relayers, e.g. an EOA to help facilitate transactions from a smart-contract wallet.
 
-Accounts in zkSync Era can initiate transactions, like an EOA, but can also have arbitrary logic implemented in them, like a smart contract. This feature is called "account
-abstraction" and it aims to resolve the issues described above.
-
-::: warning Unstable feature
-
-This is the test release of account abstraction (AA) on zkSync Era. We are very happy to hear your feedback! Please note: **breaking changes to the API/interfaces required for AA should be anticipated.**
-
-zkSync Era is one of the first EVM-compatible chains to adopt AA, so this testnet is also used to see how "classical" projects from EVM chains can coexist with the account abstraction feature.
+Accounts in zkSync Era can initiate transactions, like an EOA, but can also have arbitrary logic implemented in them, like a smart contract. This feature, called "account abstraction" (AA), aims to resolve the issues described above.
 
+::: warning
+- zkSync Era is one of the first EVM-compatible chains to adopt account abstraction. We are using the testnet to see how other projects on EVM chains coexist with account abstraction.
```

Comment:
Meaning was changed a little bit (and the old version is not wonderful as well)

"zkSync Era is the first EVM-compatible chains to implement native account abstraction. We are evaluating both on testnet and mainnet how this implementation adheres to EVM projects. No incompatibilities have been found so far."

but pls rewrite for better english and clarity

Reply:
Can you use the suggestion tool @githubdoramon ?

Reply:
DONE